[CK TILE GEMM] Refactor block_scale_gemm examples (#3181)
ROCm/composable_kernel: example/ck_tile/38_block_scale_gemm/README.md

## example
```
args:
             -h    Print help message (default:false)
             -m    m dimension (default:3840)
             -n    n dimension (default:4096)
             -k    k dimension (default:2048)
      -a_layout    A tensor data layout - Row or Column (default:R)
      -b_layout    B tensor data layout - Row or Column (default:C)
     -bq_layout    Bq tensor data layout - Row or Column (default:C)
      -c_layout    C tensor data layout - Row or Column (default:R)
      -stride_a    Tensor A stride (default:0)
      -stride_q    Tensor AQ stride (default:0)
      -stride_b    Tensor B stride (default:0)
      -stride_c    Tensor C stride (default:0)

>              -v    0. No validation, 1. Validation on CPU, 2. Validation on GPU (default:1)

Reviewer: For other options, the values and their descriptions are separated by colons. 

It should be this way here, too, not only for consistency, but also because this looks like an enumeration of options rather than a list of options and their descriptions.

```suggestion
             -v    0: No validation, 1: Validation on CPU, 2: Validation on GPU (default:1)
```

Author: Updated

>           -prec    Data type. For AQuant: fp8/bf8/i4fp8/i4bf8, For Bquant: fp8/bf8/fp8i4/bf8i4 (default:fp8)

Reviewer: ```suggestion
          -prec    Data type. For AQuant: fp8, bf8, 4fp8, or i4bf8;  for Bquant: fp8, bf8, fp8i4, or bf8i4 (default for both AQuant and Bquant: fp8)
```

The capitalization in AQuant and Bquant seem off. Is that how they're supposed to be capitalized?

Author: Updated

>         -warmup    Number of iterations before benchmark the kernel (default:50)

Reviewer: ```suggestion
        -warmup    Number of iterations before benchmarking the kernel (default:50)
```

Author: Updated
